docs/README.md

```diff
@@ -0,0 +1,208 @@
+# Usage
+
+Complete instructions for how to use the **Trustpilot** module and `trustpilot_api_client` commandline tool.
+
+## Installation
+
+Install the package from [PyPI](http://pypi.python.org/pypi/) using [pip](https://pip.pypa.io/):
+
+```
+pip install trustpilot
+```
+
+## Getting Started
+
+This client is using the [Requests](http://docs.python-requests.org/en/master/) library. Responses are standard [`requests.Response`](http://docs.python-requests.org/en/master/api/#requests.Response) objects. You can use it as a factory or as a singleton.
+
+### Use the singleton session
+
+Use the built-in `default session` to instantiate a globally accessible session.
+
+```python
+from trustpilot import client
+client.default_session.setup(
+    api_host="https://api.trustpilot.com",  # optional, default: https://api.tp-staging.com
+    api_version="v1",  # optional, default: v1
+    api_key="YOUR_API_KEY",  # required
```

Comment:
Small thing but perhaps the api_key can be first in this list to put emphasis on the fact that it is required.

Reply:
good idea